### PR TITLE
Tools 2767 support sindex on blob bins

### PR DESCRIPTION
**Pull request by dwelch-spike**

### CODE REVIEW

**Comment by dwelch-spike** [comment]
working on test fix...

**Comment by jdogmcsteezy** [comment]
I would follow the server convention in terms of branch names. That is what I did except I used bugfix instead of hotfix since that is how we refer to it. You should use `bugfix/4.1`. It is nice in some GUIs because it will actually organize it into a folder structure

image

File: .github/workflows/backup-build.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ name: Build and Test All
 
 on:
   push:
-    branches: [ master, test-ready ]
+    branches: [ master, test-ready, "bugfix-*" ]
```
Comment (jdogmcsteezy) [inline .github/workflows/backup-build.yml]: The `bugfix-*` looks like a duplicate of what I use in asadm and maybe uda/aql. If it is, I don't normally use them in this way. Normally I am not merging into bugfix but merging bugfix into master. I did not do that this time though

File: .github/workflows/mac-artifact.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 name: Mac Artifact
 on:
   push:
-    branches: [ master, test-ready, "bugfix-*", actionsHub ]
+    branches: [ master, test-ready, "bugfix-*" ]
   pull_request:
-    branches: [ master, test-ready ]
+    branches: [ master, test-ready, "bugfix-*" ]
```
Comment (jdogmcsteezy) [inline .github/workflows/mac-artifact.yml:6]: I would just check that the builds are not happening twice.

File: README.md [hunk 3]
```diff
@@ -317,7 +317,7 @@ Let's look at the placeholders, there are quite a few.
 | `{name}`       | The name of the index. |
 | `{index-type}` | The type of index: `N` = index on bins, `L` = index on list elements, `K` = index on map keys, `V` = index on map values |
 | `{path}`       | The bin name |
-| `{data-type}`  | The data type of the indexed value: `N` = numeric, `S` = string |
+| `{data-type}`  | The data type of the indexed value: `N` = numeric, `S` = string, `G` = geo2dsphere, `B` = bytes/blob, `I` = invalid |
```
Comment (jdogmcsteezy) [inline README.md:320]: When would it be invalid?

**Review by jdogmcsteezy — APPROVED** [review]
Couple comments but LGTM

File: README.md (hunk 3 above)
Comment (dwelch-spike) [reply README.md:320]: This shouldn't ever happen, but it's in the code as a default case.

**Comment by dwelch-spike** [comment]
> I would follow the server convention in terms of branch names. That is what I did except I used bugfix instead of hotfix since that is how we refer to it. You should use `bugfix/4.1`. It is nice in some GUIs because it will actually organize it into a folder structure
> image

I got this from https://aerospike.atlassian.net/wiki/spaces/TOOLS/pages/2793472010/Tools+Development+Git+Workflow where we say to use "bugfix-x.y.z" where x.y.z is the version the bugfix will be released in. I'm fine changing this though, I'll change it in this doc to be "bugfix/x.y.z" where x.y.z is the version being patched.

File: .github/workflows/mac-artifact.yml (hunk 2 above)
Comment (dwelch-spike) [reply .github/workflows/mac-artifact.yml:6]: I don't see duplicate builds in the actions run logs

File: .github/workflows/backup-build.yml (hunk 1 above)
Comment (dwelch-spike) [reply .github/workflows/backup-build.yml]: Yeah I'll remove this so only PRs will trigger the runs

**Pull request merged**